samples/SunburstChartExample/Program.cs

```diff
@@ -0,0 +1,10468 @@
+﻿using System;
+using System.IO.Packaging;
+using DocumentFormat.OpenXml;
+using DocumentFormat.OpenXml.Packaging;
+using DocumentFormat.OpenXml.Presentation;
+using A = DocumentFormat.OpenXml.Drawing;
+using P15 = DocumentFormat.OpenXml.Office2013.PowerPoint;
+using P14 = DocumentFormat.OpenXml.Office2010.PowerPoint;
+using C = DocumentFormat.OpenXml.Drawing.Charts;
+using C14 = DocumentFormat.OpenXml.Office2010.Drawing.Charts;
+using C15 = DocumentFormat.OpenXml.Office2013.Drawing.Chart;
+using C16 = DocumentFormat.OpenXml.Office2016.Drawing.ChartDrawing;
+using Cs = DocumentFormat.OpenXml.Office2013.Drawing.ChartStyle;
+using Thm15 = DocumentFormat.OpenXml.Office2013.Theme;
+using Op = DocumentFormat.OpenXml.CustomProperties;
+using Vt = DocumentFormat.OpenXml.VariantTypes;
+using Ap = DocumentFormat.OpenXml.ExtendedProperties;
+
+namespace sunburstppt
+{
+    class Program
```

Comment:
This is a very large file. Can you pare it down to a minimal viable sample? Something to show how to use it, but isn't so large?

Reply:
I reduced the sample code quite a bit although probably over time it can be reduced further.